src/events/2020-11-papercuts/index.md

```diff
@@ -48,10 +48,10 @@ Here are links to the issues identified as potential targets for papercuts day:
 
 **Europe-Middle East-Africa**
 
-* 21:00 (Melbourne, UTC +11), 09.00 (CEST, UTC +2) **Zoom Link**
-* 06:00 (EDT, UTC -4), 12.00 (CEST, UTC +2) **Zoom Link**
-* 09:00 (EDT, UTC -4), 15.00 (CEST, UTC +2) **Zoom Link**
-* 12:00 (EDT, UTC -4), 18.00 (CEST, UTC +2) **Zoom Link**
+* 21:00 (Melbourne, UTC +11), 09.00 (CEST, UTC +2) [**Zoom Link**](https://us02web.zoom.us/j/87522083149?pwd=YlRQYnVraTV6NkpaSHFvNCtwNVgxdz09)
```

Comment:
I am confused about the time zones now, this should be CET and not CEST I think

Reply:
I am always confused by this, which is one reason why I ever specify "Daylight" or "Standard" time.  (This works in the US - not so well elsewhere.)